templates/aws-stack.yml

```diff
@@ -949,10 +941,8 @@ Resources:
         Statement:
           - Effect: Allow
             Principal:
-              Service:
-                - lambda.amazonaws.com
-            Action:
-              - sts:AssumeRole
+              Service: lambda.amazonaws.com
```

Comment:
These are minor style changes that helped me read it, I’m happy to remove them to minimise the diff